src/scrollMenu.tsx

```diff
@@ -967,6 +959,13 @@ export class ScrollMenu extends React.Component<MenuProps, MenuState> {
     });
   }
 
+  /** wrapper for handleDrag event to avoid memory leak */
+  public handleDragWrapper = (
+    e: Event,
```

Comment:
```suggestion
    e: React.MouseEvent | React.TouchEvent | Event,
```

Reply:
"React.MouseEvent | React.TouchEvent" it was deleted intentionally because handleDragWrapper called only by document.addEventListener, not by react